packages/loader/container-loader/src/container.ts

```diff
@@ -1738,25 +1748,17 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         return this._deltaManager.submit(type, contents, batch, metadata);
     }
 
-    private processRemoteMessage(message: ISequencedDocumentMessage): IProcessMessageResult {
+    private processRemoteMessage(message: ISequencedDocumentMessage) {
         const local = this.clientId === message.clientId;
 
         // Allow the protocol handler to process the message
-        let result: IProcessMessageResult = { immediateNoOp: false };
-        try {
-            result = this.protocolHandler.processMessage(message, local);
-        } catch (error) {
-            this.close(wrapError(error, (errorMessage) =>
```

Comment:
Should we remove this handler? If we do, I believe the DeltaManager would catch this and still close the container on the error event from the delta queue with a DataProcessingError. But I'm not sure if that behavior has test coverage.

Reply:
I find it odd that we special case protocol failures. It does not matter where in processing pipeline error happens; any error will result in container being closed - no need for extra code (plus more chances that we will see a problem if it exists on this path)

Reply:
> any error will result in container being closed

That's true _right now_. Wasn't always the case 😁